src/plugins/workspace/public/components/workspace_detail/data_source_connection_table.tsx

```diff
@@ -174,27 +182,26 @@ export const DataSourceConnectionTable = ({
       },
     },
     {
-      width: '10%',
+      width: '15%',
```

Comment:
@Kapian1234 Seems we are re-use the same component with workspace detail side. Could you help check with @yubonluo if this width updates can be applied in workspace detail? I'm prefer to add `flag` here to determine if we need to update the width.

Reply:
The width updates will indeed be applied in workspace detail, but I've checked with @yubonluo and the new width still presents well there. Perhaps the flag isn't necessary?